src/data/complex/exponential.lean

```diff
@@ -1630,6 +1630,17 @@ end
 
 end real
 
+namespace tactic
+open positivity real
+
+/-- Extension for the `positivity` tactic: `real.exp` is always positive. -/
+@[positivity]
+meta def positivity_exp : expr → tactic strictness
+| `(exp %%a) := positive <$> mk_app ``exp_pos [a]
```

Comment:
```suggestion
| `(exp %%a) := positive <$> mk_app `real.exp_pos [a]
```
perhaps I'm being overly paranoid here, but if there's something else in the local context called `exp_pos` this application might fail (eg matrix exponential positivity, or something conditional with the complex exponential). Probably this is unlikely, but we might as well be a bit more robust